strr-api/devops/gcp/clouddeploy.yaml

```diff
@@ -84,5 +89,8 @@ serialPipeline:
       container-name: "strr-api-prod"
       cloudsql-instances: "bcrbk9-prod:northamerica-northeast1:strr-db-prod"
       service-account: "sa-api@bcrbk9-prod.iam.gserviceaccount.com"
-      max-scale: "10"
-      container-concurrency: "20"
+      max-scale: "30"
+      container-concurrency: "80"
```

Comment:
Probably start closer to 8 than 80 https://cloud.google.com/run/docs/about-concurrency#source